packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -328,7 +328,7 @@ function useFocusContainment(scopeRef: RefObject<Element[] | null>, contain?: bo
 
     // Handle the Tab key to contain focus within the scope
     let onKeyDown = (e) => {
-      if (e.key !== 'Tab' || e.altKey || e.ctrlKey || e.metaKey || !shouldContainFocus(scopeRef) || e.isComposing) {
+      if (e.key !== 'Tab' || e.altKey || e.ctrlKey || e.metaKey || e.target.tagName === 'VIDEO' || !shouldContainFocus(scopeRef) || e.isComposing) {
```

Comment:
Wouldn't this break containment if it is the first or last element in the scope? You'd be able to tab out of the scope then.

Reply:
I think it might actually result in a circle. The video appears to receive focus again and then you go through the controls again, then it goes back to the video. Not sure why.
https://github.com/adobe/react-spectrum/issues/6729#issuecomment-2240518557